doc/02-installation.md.d/01-Debian.md

```diff
@@ -1,3 +1,3 @@
-# Install Icinga 2 on Debian
+# Install Icinga 2 on Debian and Raspberry Pi OS
```

Comment:
If that page is rendered twice, is there a reason for saying "Debian and Raspberry Pi OS" on both?